internal/operator/namespace.go

```diff
@@ -34,21 +32,21 @@ func CreateNamespace(ctx context.Context, name string) (*corev1.Namespace, error
 	}
 	_, err := operatorClient.CoreV1().Namespaces().Create(ctx, &nsSpec, metav1.CreateOptions{})
 	if err != nil {
-		log.Debug(fmt.Errorf("%w: error while creating Namespace: %s", err, name))
+		logger.Errorf("error while creating Namespace %s: %s", name, err)
 		return nil, err
 	}
-	log.Debugf("Namespace Created: ", name)
+	logger.Infof("Namespace Created: %s", name)
```

Comment:
This change will introduce a lot of data on the screen. We need a way to separate Debug from Info. This is definitely something that user doesn't need to know or doesn't want to see. We need to have the progress on the operator tests as clean as possible on the screen.